packages/core/src/components/select/select.scss

```diff
@@ -53,26 +75,60 @@
         color: helpers.color('content-disabled');
       }
     }
+  }
 
-    &.-borderless {
-      @include mixins.with-inner-focus('action');
-      @include with-padding(); // no border width because border will be unset
+  .ods-select-custom.-open + .ods-icon {
+    transform: rotate(180deg);
+  }
 
-      background-color: transparent;
-      border: 0;
+  .ods-select-custom-dropdown {
+    @include mixins.with-inner-focus('action');
```

Comment:
The "border" being our general "on focus inset box-shadow" can be a problem:

![image](https://user-images.githubusercontent.com/18623773/123136979-6d84fd80-d44b-11eb-9135-a7d1ff2a5dc1.png)

Granted, the dropdown should probably not be visible in the case when it's not focused.

So although it makes it DRY to reuse our mixin, I believe it works better as an actual border here.

Also, I wonder if a tiny "box-shadow" for a sense of elevation would be good.

Reply:
Probably that's the case - I will move from using a mixin to just adding to a dropdown element itself.

I do not want to add elevation here, only because elevation tokens are to be still added by @Subcide, so we can add at a later stage as component improvement.